plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py

```diff
@@ -143,5 +143,75 @@ def test_sessionLayerSerialisation(self):
         os.remove(_tmpMayafile.name)
 
 
+    def test_multipleFormatsSerialisation(self):
+        """Tests multiple dirty layers with various formats can be saved and restored.
+
+        """
+        _, rootLayerPath = tempfile.mkstemp(suffix=".usda")
+        _, tmpMayafilePath = tempfile.mkstemp(suffix=".ma")
```

Comment:
We have moved away from using temp files, because:
* it's harder to debug
* required periodic temp folder cleanup

For this reason, we use explicit names and local to the test area within the build folder. Generated files are easy to discover and the build folder is more often cleared than the temp folder.

Here is an example - https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/translators/testUsdExportMaterialX.py#L43

Reply:
Hi @kxl-adsk, looks like PF checks are now passing, and I've updated the test to avoid tempfiles (we'll keep this in mind going forward). Let me know if there's anything else needed.